### ISSUE TITLE
Finalization of survey launch 

### ISSUE THREAD

**Issue by rlonka** [nordic-rse/RSE_intro_survey#11]
I think we have just few points left:

1. Finalize email text (it looks very good now, so please validate)
2. agree launch day and when it will end (are we able to finish before  RSE meeting/sept.?)
3. test it (I guess we can be first to fill it then we will spread it)
4. create short url, read @bast [issue comment](https://github.com/nordic-rse/RSE_intro_survey/issues/12#issuecomment-383524289)
4. any other point?

**Comment by rlonka** [comment]
Also I would ask everyone to test the survey, we will delete all data before we launch it

**Comment by rlonka** [comment]
1. I guess we done...
2. if you give me green from at least 2 people I will send you all the beta survey version
3. based on point 2, I would ask you to test  the survey, I will remove the data  before we launch it
4. After point 3 I will make a short url and send you all official version

Is that ok?

**Comment by wikfeldt** [comment]
i tested the survey, looks good to me

**Comment by bast** [comment]
https://docs.google.com/forms/d/e/1FAIpQLSdOqNU8ENHRaJ7K5eYn_18Brx3RzdoxC5IuGnq1zUeaRGD7AQ/viewform

**Comment by bast** [comment]
Looks good to me, too.

**Comment by rlonka** [comment]
How about the end of survey? I thought mid of August. Then we will have 2-3 weeks to analyze data and compare with other RSE groups.
It would seems better until end of August but I am not sure if we will have enough of time for analyzing

**Comment by bast** [comment]
Why not much earlier? How much time does the survey need? People who don't fill it out within 5 minutes after reading the email probably never will. Reminders take some time, also forwarding to different places takes time but I see no problem in having results in before August.

**Comment by rlonka** [comment]
I just though to have it as long as possible to gather more data. But I do not have any strong opinion and neither experience... So what would you  suggest @bast ?

**Comment by wikfeldt** [comment]
do we absolutely need to decide on an end date? if the turnout turns out to be very low we might want to keep it open longer, and vice versa.

**Comment by rlonka** [comment]
that's right, we do not need the end date after all. First I thought it would be to inform user when she/he can expect result, but if they will follow twitter/mailinglist/webpage they will get informed anyway.
So no end date will be provided!

**Comment by rlonka** [comment]
there was an idea of @bast to provide shorter better url (via bitly), how about to use our domain and create http://nordic-rse.org/survey2018 which redirect to the from?

**Comment by wikfeldt** [comment]
or we use http://www.shadyurl.com/ for a more dodgy-looking url
;-)

no but it could look good to have our domain, but I don't see it as super important.

**Comment by rlonka** [comment]
I did not know the shadyurl! Looks pretty cool :smile: , Yes, lets sabotage the survey! :laughing:

**Comment by bast** [comment]
It's not ultra important. I think own domain redirecting to the form is also really a good idea. I like if I see from the URL whether "I can trust it".
